packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -600,11 +755,16 @@ class ScaffoldPrelayoutGeometry {
   /// appropriate insets to apply.
   final Size scaffoldSize;
 
-  /// The [Size] of the [Scaffold]'s [SnackBar].
+  /// The [Size] of the [Scaffold]'s [MaterialBanner].
   ///
-  /// If the [Scaffold] is not showing a [SnackBar], this will be [Size.zero].
+  /// If the [Scaffold] is not showing a [MaterialBanner], this will be [Size.zero].
   final Size snackBarSize;
```

Comment:
Is this snack bar or banner here? The docs are a bit confusing.

Reply:
snackbar, corrected